Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -248,11 +251,42 @@ static VertexLoaderBase* RefreshLoader(int vtx_attr_group, bool preprocess = fal
   return loader;
 }
 
-int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bool is_preprocess)
+void UpdateCyclesPerVertex()
```

Comment:
What do you think about caching the cycles_per_vertex as attribute of our vertex loaders?